L1Trigger/Phase2L1ParticleFlow/python/l1pfProducer_cfi.py

```diff
@@ -18,9 +19,9 @@
      puppiEtaCuts       = cms.vdouble(1.5, 2.5, 3.0, 5.5), 
      puppiPtCuts        = cms.vdouble(0.0, 3.0, 6.0, 8.0),
      puppiPtCutsPhotons = cms.vdouble(0.0, 3.0, 6.0, 8.0),
-     vtxRes      = cms.double(0.333),
+     vtxRes      = cms.double(0.15),
      vtxAlgo     = cms.string("TP"),
```

Comment:
if reading the PV externally we no longer need the "vtxAlgo" parameter, it can be removed to avoid confusion